star-war-character-app/src/redux/PeopleSlice/PeopleType.tsx

```diff
@@ -0,0 +1,26 @@
+export interface Peoples {
+    url: string;
+    id: number;
+    name: string;
+    height: number;
+    mass: number;
+    hair_color: string;
+    skin_color: string;
+    eye_color: string;
+    birth_year: string;
+    gender: string;
+    homeworld: string;
+    films: string[];
+    species: string[];
+    created: string;
+    edited: string;
+  }
+  export interface PeoplesList {
+    list: Peoples[];
```

Comment:
It should be `People`

Reply:
I have resolved this issue in this PR https://github.com/Harsh-Upadhyay-AT/Star-War-Character-App/pull/2 Please check it.